### ISSUE TITLE
Serializing ListResult results in Exception

### ISSUE THREAD

**Issue by vdurante** [chargebee/chargebee-dotnet#44]
Apparently the server is not returning a `subscription` property called `has_scheduled_advance_invoices`. The serializer tries to read its value, which runs this code, which causes an exception: https://github.com/chargebee/chargebee-dotnet/blob/031f69b924a7d0a6d3d33745707c79138978b84d/ChargeBee/Models/Subscription.cs#L372

**Comment by ThirstyJarek** [comment]
I'm also receiving this same error when attempting to serialise a subscription object to json.

For example, this code fails:
`var serialisedJson = JsonConvert.SerializeObject(chargebeeSubscription)`

With this error:
"Error getting value from 'HasScheduledAdvanceInvoices' on 'ChargeBee.Models.Subscription'."

**Comment by remcolam** [comment]
3 months later, this issue still persists. As a workaround, I created the following extenstion method:
```
	public static T GetFieldValue<T>(this object businessObject, string fieldName, T defaultValue = default, BindingFlags bindingFlags = BindingFlags.Default)
	{
		return (T)(businessObject.GetType().GetField(fieldName, bindingFlags)?.GetValue(businessObject) ?? defaultValue);
	}
```

Then, to serialize the response, I do the following (yeah, very evil and ugly, but it works):
```
				var response = Newtonsoft.Json.JsonConvert.SerializeObject(new Dictionary<string, object>
				{
					{ nameof(entityListResult.StatusCode), entityListResult.StatusCode },
					{ nameof(entityListResult.List), entityListResult.List?.Select(i => i.GetFieldValue<Newtonsoft.Json.Linq.JToken>("m_jobj", bindingFlags: System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.NonPublic)) },
				});
```

**Comment by mcelligottnick** [comment]
Still getting this error on 2.17.0. Downgraded to 2.7.7 and not getting this error anymore.

**Comment by mcelligottnick** [comment]
Yep this happened in https://github.com/chargebee/chargebee-dotnet/releases/tag/v2.8.7, so anything below that will work for now.

**Comment by dkattan** [comment]
Any plans to fix this?

**Comment by cb-khushbubibay** [comment]
Hi All,
Thanks for raising this concern.
This issue has been fixed in v2.19.0 version, requesting you all to please update the package and tried once.
